runtime/moonbase/src/lib.rs

```diff
@@ -1126,6 +1125,113 @@ where
 	}
 }
 
+// Name of the pallet to be removed in the migration
+// The pallet name is used as storage key prefix for all the values stored by pallets
+frame_support::parameter_types! {
+	pub const LocalAssetsPalletName: &'static str = "LocalAssets";
+}
+
+pub struct RemoveLocalAssets<R>(sp_std::marker::PhantomData<R>);
```

Comment:
Please move the migration inside a migrations.rs file in the moonbase-runtime crate

Reply:
Just for clarity, there were other migrations in the `lib.rs` file. I will move all of them into a migrations file

Reply:
Removed the migration, instead will include a new extrinsic for pallet storage removal in a new PR.